published/Daily_goals_and_results/actions/ask-me-how-my-day-went/triggers.json

```diff
@@ -0,0 +1,6 @@
+[ {
+  "text" : "ask me how my day went",
+  "requiresMention" : false,
```

Comment:
Should we change all these action triggers to requiresMention: true in keeping with our recent discussion?